app/admin/export/template.hbs

```diff
@@ -3,4 +3,8 @@
     <h1>Export</h1>
   </div>
   {{data-export range=model}}
```

Comment:
Let's rename this to `jobs-export` or `open-jobs-export` to better reflect what it does and make it consistent with other component names.